client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -311,19 +312,19 @@ const LowerJaw = ({
           {t('learn.sign-in-save')}
         </a>
       )}
-      <button
+      <Button
         data-playwright-test-label='lowerJaw-submit-button'
-        className='btn-block btn'
+        block
         data-cy='submit-lowerJaw-button'
         onClick={tryToSubmitChallenge}
         {...(!challengeIsCompleted && { 'aria-hidden': true })}
         ref={submitButtonRef}
       >
         {t('buttons.submit-and-go')}
-      </button>
-      <button
+      </Button>
+      <Button
         data-playwright-test-label='lowerJaw-check-button'
-        className='btn-block btn'
+        block
         data-cy='check-lowerJaw-button'
         onClick={tryToExecuteChallenge}
         {...(challengeIsCompleted &&
```

Comment:
Because we are now using the `<Button>` component, it brings a lot of default classNames that I couldn't find a way to overwrite them just like previously so it only has the `sr-only` class when changing, it ends up only appending the class with the rest.

Because of this, when submitting the challenge, an empty button makes it appearance. There are three ways that I think it could be fixed, but correct me if I'm wrong or is too much overhead:

1. Force a class (like `.sr-only`) with an `!important` to make the button disappear, not a fan of it.
2. Add an `overrideClassName` prop to the `Button` component, so the previous behavior can be replicated, and pass the `sr-only` className like it's done now. I think this could be the best option.
3. A third accessible option that I'm not aware of. 

Any other suggestions are welcomed.

https://github.com/freeCodeCamp/freeCodeCamp/assets/47751447/194b4130-db77-46a1-843d-d15786dc8fbd

Reply:
My b, didn't select the correct lines. Referring to lines 330 and 331.